cmd/podman/common/specgen.go

```diff
@@ -655,26 +655,53 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *ContainerCLIOpts, args []string
 	return nil
 }
 
-func makeHealthCheckFromCli(inCmd, interval string, retries uint, timeout, startPeriod string) (*manifest.Schema2HealthConfig, error) {
+func makeHealthCheckFromCli(inCmd string, interval string, retries uint, timeout, startPeriod string) (*manifest.Schema2HealthConfig, error) {
+	cmdArr := &[]string{}
```

Comment:
The double-indirection here is kind of gross - would be better to specify as `[]string{}` and then add the `&` only to the `json.Unmarshal` call below.